### DISCUSSION TITLE
Speed Editor support without Resolve running?

### DISCUSSION THREAD

**Discussion by davidgiven** [CommandPost/CommandPost#3355] [General]
I don't have a Mac, so can't test this myself, but:

Does CommandPost support the Speed Editor without having Resolve running in the background? If so, how does CommandPost manage to unlock the Speed Editor to make it produce hid events? From looking through the codebase I can't see any auth code for doing the unlock, so I'm a bit puzzled.

(Context: I'd like to use the Speed Editor from a platform which CommandPost doesn't support and am looking to see what's required to wake it up.)

Thanks!

**Comment by randomeizer** [comment]
I believe the actual connection code is here:

https://github.com/CommandPost/CommandPost-App/tree/master/extensions/blackmagic

From memory, that's because it has C code needing to be compiled, and it was easier to set up there.

**Comment by davidgiven** [comment]
That's great --- thanks!

I'll report back if I ever get it working. Although if you're planning on doing a port of CommandPost to Linux, that'd be nice too...

**Reply by randomeizer** [reply]
It's pretty unlikely I'm afraid. The core app CP is built on (Hammerspoon) is macOS only, and Chris and I don't use Linux.

Would love to hear about your progress though.
